src/polyfill/chunks_fixed.rs

```diff
@@ -27,9 +27,5 @@ macro_rules! define_chunks_fixed {
 }
 
 // Sorted by the first value, then the second value.
-define_chunks_fixed!(12, 4);
-define_chunks_fixed!(16, 4);
-define_chunks_fixed!(16, 8);
-define_chunks_fixed!(32, 4);
```

Comment:
For the small arrays that are converted here (the ones for which lines are removed above) this approach seems to optimize well. For the other two larger array sizes (mentioned below), I will post a separate PR that uses different techniques.